### DISCUSSION TITLE
Not allowing qty zero on component

### DISCUSSION THREAD

**Discussion by Bowen73** [snipe/snipe-it#10324] [Q&A]
When we try and set a component as zero qty it throws up an error to say the qty cannot be zero.  but if there is no stock then the stock level will be wrong as its showing a quantity of something we don't have.

how can I get it so we can set the qty to zero?

thanks

**Comment by 3nv1r0** [comment]
Same question here. We would like to use Snipe-IT to manage our IT department stock and zero quantity would mean that the asset or consumable is currently out of stock. Is there any way to change that behavior?
